provisioner/windows-restart/provisioner.go

```diff
@@ -183,12 +189,20 @@ var waitForCommunicator = func(p *Provisioner) error {
 		log.Printf("Checking that communicator is connected with: '%s'", cmd.Command)
 
 		err := cmd.StartWithUi(p.comm, p.ui)
+
 		if err != nil {
 			log.Printf("Communication connection err: %s", err)
 			continue
 		}
 
 		log.Printf("Connected to machine")
+		stdoutToRead := buf2.String()
+		buf2.Reset()
```

Comment:
I don't _think_ these resets are needed since the buf vars get redeclared at the top of the loop

Reply:
ah, you're probably right.  I initially had the buffer variables declared outside the loop.